unime/src/i18n/en/index.ts

```diff
@@ -189,26 +194,64 @@ const en = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
+      NAVBAR_TITLE: 'Credential info',
+      NAME: '',
+    },
+    BADGE: {
+      NAVBAR_TITLE: 'Badge info',
+      DESCRIPTION: 'Description',
+      METADATA: 'Metadata',
+    },
     EMPTY_CREDENTIALS: {
       TITLE: "It's a bit quiet in here",
       SUBTITLE: 'Why not add some credentials to start your new digital me?',
     },
     FAVORITES: 'My favorites',
   },
   ACTIVITY: {
+    BOTTOM_NAVIGATION_TITLE: 'Activity',
     NAVBAR_TITLE: 'Connected',
     TABS: {
       CONNECTIONS: 'Connections',
       TIMELINE: 'Timeline',
     },
   },
+  SCAN: {
+    BOTTOM_NAVIGATION_TITLE: 'Scan',
+    TITLE: 'Scan a',
+    TITLE1: 'QR Code',
+    SUBTITLE: 'Bring a QR Code into view of this screen to start an interaction.',
+    PERMISSION: 'No permissions to',
```

Comment:
Please follow the pattern `PERMISSION_1`, `PERMISSION_2` for multiline text (instead of `PERMISSION`, `PERMISSION1`).

Reply:
We should update to `NO_PERMISSION_1`, `NO_PERMISSION_2` to make it even clearer that is only shown when there is _no_ permission given.